src/components/Settings.js

```diff
@@ -70,6 +92,71 @@ class Settings extends Component {
           />
           <span>Oversample</span>
         </div>
+
+        <hr />
```

Comment:
I like the inclusion of the `<hr />` tag, but stylistically, it looks a bit improved (less squished) with a margin of 1rem set to it.

Reply:
Done